Kuryr: Add a 4.5 known issue (#34557)
openshift/openshift-docs: release_notes/ocp-4-5-release-notes.adoc

// TODO: This known issue should carry forward to 4.6 and beyond!
* The `oc annotate` command does not work for LDAP group names that contain an equal sign (`=`), because the command uses the equal sign as a delimiter between the annotation name and value. As a workaround, use `oc patch` or `oc edit` to add the annotation. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917280[*BZ#1917280*])

> * kuryr-controller may crash if {product-title} cluster includes Services having all member pods in not ready state. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1980957[*BZ#1980957*])

Reviewer: ```suggestion
* Instances of kuryr-controller might crash if the cluster includes services for which all member pods are not ready. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1980957[*BZ#1980957*])
```

Author: Awesome, I force-pushed your version.
